demo/Script/FmodTest.gd

```diff
@@ -1,21 +1,3 @@
 extends Node
 
-# Called when the node enters the scene tree for the first time.
-func _enter_tree():
-	# set up FMOD
-	Fmod.set_software_format(0, Fmod.FMOD_SPEAKERMODE_STEREO, 0)
-	Fmod.init(1024, Fmod.FMOD_STUDIO_INIT_LIVEUPDATE, Fmod.FMOD_INIT_NORMAL)
-	Fmod.set_sound_3D_settings(1, 32, 1)
-	Fmod.set_listener_number(2)
-	Fmod.set_default_callback(Fmod.FMOD_STUDIO_EVENT_CALLBACK_ALL)
```

Comment:
set_default_callback is still necessary in the demo. As it is right now, the beat signal is not received anymore ( top right square is supposed to change color.
You have to add that line again somewhere in the demo project.

Reply:
That function seemed to missing from `Fmod.gd` on my fork and when I tried adding it in, I got this error 
![image](https://user-images.githubusercontent.com/48840438/204757008-56871260-3989-473d-8c5b-8bc1e621e7ea.png)

I coppied this from the current repository where it can be found
![image](https://user-images.githubusercontent.com/48840438/204757080-bc8fd11e-b726-473c-ba3d-26f64d95e0c4.png)

Reply:
you should rebase your branch on the current master to find it.

Reply:
same here, you changed those parts:
<img width="768" alt="image" src="https://user-images.githubusercontent.com/31470327/204758804-f56f530b-142d-43e2-a333-62b04d3e30b1.png">

Reply:
set_default_callback was added in master recently so maybe you just didn't update your fork with the more recent changes. You should rebase your branch on master.

Reply:
deleting that was a mistake. Thanks for letting me know. I'll rebase my fork now.